Libraries/Text/TextInput/RCTBackedTextInputDelegateAdapter.m

```diff
@@ -217,7 +217,6 @@ - (BOOL)control:(NSControl *)control textView:(NSTextView *)fieldEditor doComman
     //escape
   } else if (commandSelector == @selector(cancelOperation:)) {
     [textInputDelegate textInputDidCancel];
-    [[_backedTextInputView window] makeFirstResponder:nil];
```

Comment:
We can make it work - but only if ```validKeysUp|Down``` is not set.
Otherwise that would take precedence.

At least the test case for https://github.com/microsoft/react-native-macos/pull/1044 still seem to work for
singleLineTextInputs

https://user-images.githubusercontent.com/484044/184236447-eb99d1a7-516c-4d2e-9ad4-aa4a7c68b6cd.mov



and multilineTextInputs


https://user-images.githubusercontent.com/484044/184236461-303ee0b7-f55e-450b-844d-2e32c3bd3c5a.mov

Reply:
OK, I highly doubt we'd want the allowlist nature of `validKeys(Up|Down)` on text input anyway, so that makes sense :)